### PR TITLE
CTPS open new SM sites

### PR DESCRIPTION
**Pull request by Thomas-Benson**
Opening new SM site `CTPS - Centreport`
 - Digitiser: Centaur (11447)
   - Centaur firmware version 4.8.5
 - LC 20: Titan TACCL-N1 (3397)
   - Sensor located at surface
   - Streaming on LC20 HN-ENZ
 - LC 21: Titan TACCL-PH2 (3526)
   - Sensor located in borehole at a depth of  33m
   - Streaming on LC21 HN-12Z

TODO: 
Check Titan TACCL-PH2 response
Confirm orientation
Get correct starttime - done

### CODE REVIEW

**Comment by Thomas-Benson** [comment]
Looking at the data we want to backfill, it looks like the final starttime for this site will be `2024-09-23T00:00:00Z`

File: install/sensors.csv [hunk 1]
```diff
@@ -818,7 +818,9 @@ Licor,Li-850 CO2/H2O Gas Analyzer,HGA-06919,TO006,05,0,,0,0,0,0,0,0,2024-06-04T0
 Licor,Li-850 CO2/H2O Gas Analyzer,HGA-06920,TO003,12,0,,0,0,0,0,0,0,2024-03-01T00:20:00Z,9999-01-01T00:00:00Z
 Nanometrics,Titan TACCL-BH1,2363,CBGS,22,0,,0,21.8,0,0,0,0,2023-02-23T03:16:00Z,9999-01-01T00:00:00Z
 Nanometrics,Titan TACCL-N1,2590,CBGS,21,0,,0,0,0,0,0,0,2023-02-23T03:16:00Z,9999-01-01T00:00:00Z
+Nanometrics,Titan TACCL-N1,3397,CTPS,20,0,,0,0,0,0,0,0,2024-09-23T00:00:00Z,9999-01-01T00:00:00Z
 Nanometrics,Titan TACCL-N1,3399,SMHS,20,230,,0,0,0,0,0,0,2024-07-01T03:10:00Z,9999-01-01T00:00:00Z
+Nanometrics,Titan TACCL-PH2,3526,CTPS,21,33,,0,0,0,0,0,0,2024-09-23T00:00:00Z,9999-01-01T00:00:00Z
```
Comment (mnaguit) [inline install/sensors.csv:823]: just to double check if azimuth is `33` (it is not indicated in the ticket --or perhaps I missed that detail)

File: network/sites.csv [hunk 2]
```diff
@@ -757,6 +757,8 @@ CSWB,10,-41.57143,174.12668,268,,WGS84,External GPS Device,2013-07-23T05:20:10Z,
 CSWC,10,-41.57146,174.12672,269,,WGS84,Internal GPS Clock,2013-07-24T05:05:00Z,2013-11-19T10:00:00Z
 CSWS,20,-41.57143,174.12668,268,,WGS84,External GPS Device,2013-07-23T05:20:10Z,2013-11-19T10:00:00Z
 CTND,10,-43.47686,172.21852,170,,WGS84,External GPS Device,2010-09-05T01:35:02Z,2010-09-29T23:00:00Z
+CTPS,20,-41.278513,174.785939,5,,WGS84,Internal GPS Clock,2024-09-23T00:00:00Z,9999-01-01T00:00:00Z
+CTPS,21,-41.278513,174.785939,5,,WGS84,Internal GPS Clock,2024-09-23T00:00:00Z,9999-01-01T00:00:00Z
```
Comment (mnaguit) [inline network/sites.csv:761]: double checking  the elevation, was it mentioned in the ticket as well?

File: install/components.csv [hunk 3]
```diff
@@ -123,6 +123,9 @@ Nanometrics,Titan TACCL-BH1,Accelerometer,2,,E,0,90,G,,sensor_Nanometrics-Inc_Ti
 Nanometrics,Titan TACCL-N1,Accelerometer,0,,Z,-90,0,G,,sensor_Nanometrics-Inc_Titan-TACCL-N1
 Nanometrics,Titan TACCL-N1,Accelerometer,1,,N,0,0,G,,sensor_Nanometrics-Inc_Titan-TACCL-N1
 Nanometrics,Titan TACCL-N1,Accelerometer,2,,E,0,90,G,,sensor_Nanometrics-Inc_Titan-TACCL-N1
+Nanometrics,Titan TACCL-PH2,Accelerometer,0,,Z,-90,0,G,,sensor_Nanometrics-Inc_Titan-TACCL-N1
+Nanometrics,Titan TACCL-PH2,Accelerometer,1,,N,0,0,G,,sensor_Nanometrics-Inc_Titan-TACCL-N1
+Nanometrics,Titan TACCL-PH2,Accelerometer,2,,E,0,90,G,,sensor_Nanometrics-Inc_Titan-TACCL-N1
```
Comment (mnaguit) [inline install/components.csv:128]: shall we wait until the Titan TACCL-PH2 response is sorted?

File: install/connections.csv [hunk 4]
```diff
@@ -153,6 +153,8 @@ CSWB,10,White Bluffs,Portable,,2013-07-23T05:20:10Z,2013-11-19T10:00:00Z
 CSWC,10,White Bluffs Solar Site with Comms,,,2013-07-24T05:05:00Z,2013-11-19T10:00:00Z
 CSWS,20,White Bluffs strong-motion,,,2013-07-23T05:20:10Z,2013-11-19T10:00:00Z
 CTND,10,Courtenay,Portable,,2010-09-05T01:35:02Z,2010-09-29T00:00:00Z
+CTPS,20,Centreport,,,2024-09-23T00:00:00Z,9999-01-01T00:00:00Z
+CTPS,21,Centreport,,,2024-09-23T00:00:00Z,9999-01-01T00:00:00Z
```
Comment (mnaguit) [inline install/connections.csv:157]: Noting that if `Titan TACCL-PH2` has the same response as `TACCL-N1` , channel number/datalogger pin does not need to be updated.

File: network/sites.csv (hunk 2 above)
Comment (Thomas-Benson) [reply network/sites.csv:761]: I copied the coordinates and elevation from the locality in the network PR for this site.

File: install/connections.csv (hunk 4 above)
Comment (Thomas-Benson) [reply install/connections.csv:157]: It's been confirmed that the responses are identical

**Review by mnaguit — DISMISSED** [review]
Thanks @Thomas-Benson 👍

**Comment by Thomas-Benson** [comment]
Updated the response for the TACCL-PH2 to use the base `sensor_Nanometrics-Inc_Titan` response rather than the instrument specific `sensor_Nanometrics-Inc_Titan-TACCL-N1`. There is no change in response generated sinnce these are identical, but it provides more clarity and is more consistent with other Titan variants which all use `sensor_Nanometrics-Inc_Titan` too.

**Review by mnaguit — APPROVED** [review]

File: install/components.csv [hunk 5]
```diff
@@ -123,6 +123,9 @@ Nanometrics,Titan TACCL-BH1,Accelerometer,2,,E,0,90,G,,sensor_Nanometrics-Inc_Ti
 Nanometrics,Titan TACCL-N1,Accelerometer,0,,Z,-90,0,G,,sensor_Nanometrics-Inc_Titan-TACCL-N1
 Nanometrics,Titan TACCL-N1,Accelerometer,1,,N,0,0,G,,sensor_Nanometrics-Inc_Titan-TACCL-N1
 Nanometrics,Titan TACCL-N1,Accelerometer,2,,E,0,90,G,,sensor_Nanometrics-Inc_Titan-TACCL-N1
+Nanometrics,Titan TACCL-PH2,Accelerometer,0,,Z,-90,0,G,,sensor_Nanometrics-Inc_Titan
+Nanometrics,Titan TACCL-PH2,Accelerometer,1,,N,0,0,G,,sensor_Nanometrics-Inc_Titan
+Nanometrics,Titan TACCL-PH2,Accelerometer,2,,E,0,90,G,,sensor_Nanometrics-Inc_Titan
 Nanometrics,Trillium 120QA,Broadband Seismometer,0,,Z,-90,0,G,,sensor_Nanometrics-Inc_Trillium-120QA
```
Comment (CBurton90) [inline install/components.csv:129]: One to double check (I'll have a look as well) but are the `sensor_Nanometrics-Inc_Titan` and `sensor_Nanometrics-Inc_Titan-TACCL-N1` the same response? We could looks to do some consolidation if so.

cc @ozym

**Review by CBurton90 — APPROVED** [review]

**Review by salichon — APPROVED** [review]
looks good to me hey :)

File: install/components.csv (hunk 5 above)
Comment (Thomas-Benson) [reply install/components.csv:129]: There's six resp files for Titan sensors, and they are all identical. The versions named sensor_Nanometrics_Titan do not appear to be used anywhere that I can see either
```
sensor_Nanometrics-Inc_Titan-TACCL-BH1.xml  
sensor_Nanometrics-Inc_Titan-TACCL-N1.xml  
sensor_Nanometrics-Inc_Titan.xml  
sensor_Nanometrics_Titan-TACCL-BH1.xml  
sensor_Nanometrics_Titan-TACCL-N1.xml  
sensor_Nanometrics_Titan.xml
```

**Pull request merged**